x-pack/packages/kbn-ecs-data-quality-dashboard/impl/data_quality/data_quality_panel/index_properties/markdown/helpers.ts

```diff
@@ -244,101 +219,121 @@ export const getResultEmoji = (incompatible: number | undefined): string => {
 };
 
 export const getSummaryTableMarkdownHeader = (): string =>
-  `| ${RESULT} | ${INDEX} | ${DOCS} | ${INCOMPATIBLE_FIELDS} | ${ILM_PHASE} |
-|--------|-------|------|---------------------|-----------|`;
+  `| ${RESULT} | ${INDEX} | ${DOCS} | ${INCOMPATIBLE_FIELDS} | ${ILM_PHASE} | ${SIZE} |
+|--------|-------|------|---------------------|-----------|------|`;
```

Comment:
SUPER random thought, but you can have a space-builder so it's easier to get an idea of the numerical spacing.

Something like `${addSpace(8)}` and `const addSpace = (spaces: number) => Array(spaces).fill('-').join('');`. DEFINITELY over-engineered, but couldn't help myself